### PR TITLE
[TACHYON-273] fix copyToLocal command

### PR DESCRIPTION
**Pull request by Yours-shupeng**
We fix the usage of copyToLocal command in this way: tfs copyToLocal \<src\> \<localDst\> [-bufferSize size], size is an integer value representing number of bytes.  If we still use the default command without <-bufferSize size>, the download speed  will keep slow. While if we assign bufferSize to a large integer like 5120000, the speed will improve a lot. For example, if we download a 1G file , we can use -bufferSize 51200000.

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -139,6 +139,11 @@
       <artifactId>jackson-databind</artifactId>
       <version>2.3.0</version>
     </dependency>
+    <dependency>
+      <groupId>de.tototec</groupId>
+      <artifactId>de.tototec.cmdoption</artifactId>
```
Comment (haoyuan) [inline core/pom.xml]: why is this needed?

Comment (Yours-shupeng) [inline core/pom.xml]: This package is for CommandLineParser. If we add additional parameters like [-parameter value], we can its function conveniently.
